base/applications/mstsc/proto.h

```diff
@@ -149,7 +149,7 @@ RD_BOOL rd_lock_file(int fd, int start, int len);
 /* rdp5.c */
 void rdp5_process(STREAM s);
 /* rdp.c */
-void rdp_out_unistr(STREAM s, char *string, int len);
+void rdp_out_unistr(STREAM s, char *string, SIZE_T len);
```

Comment:
Maybe to respect the ambient code-style, "size_t" in lowercase. Also it seems that the counterpart function "rdp_in_unistr" (below) thinks that the string size will either be a uint32, or (what is this "in_len"?) an int.

Reply:
done